src/styles/index.scss

```diff
@@ -1,3 +1,56 @@
+@import '/src/styles/variables';
+
 body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $with;
+  gap: $size-gap;
+
+  &_day {
```

Comment:
There is a typo in the class name `&_day`. It should be `&_day`. According to the BEM methodology, element names are separated by two underscores. It's important to follow the BEM methodology for better code readability and maintainability.